src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/IntegrationRoadNodesToZipArchiveWriter.cs

```diff
@@ -32,21 +33,16 @@ public IntegrationRoadNodesToZipArchiveWriter(RecyclableMemoryStreamManager mana
 
         const int integrationBufferInMeters = 350;
 
-        var nodesInContour = await context.RoadNodes
-            .InsideContour(request.Contour)
-            .ToListAsync(cancellationToken);
+        var nodesInContour = await context.RoadNodes.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
         var geometryForNodesInContour = GeometryConfiguration.GeometryFactory
             .BuildGeometry(nodesInContour.Select(node => node.Geometry));
 
         var boundaryForNodes = geometryForNodesInContour.ConvexHull();
         var boundaryWithIntegrationBuffer = boundaryForNodes.Buffer(integrationBufferInMeters);
 
-        var nodesInIntegrationBuffer = await context.RoadNodes
-            .InsideContour(boundaryWithIntegrationBuffer as IPolygonal)
-            .ToListAsync(cancellationToken);
+        var nodesInIntegrationBuffer = await context.RoadNodes.ToListWithPolygonials(boundaryWithIntegrationBuffer as IPolygonal, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done